modules/dlbclass/1.0/src/sv2gsm.R

```diff
@@ -21,17 +21,27 @@ samples <- samples[[1]]
 # Load SV file
 svs <- lapply(sv_files, read.delim, sep = "\t")
 svs <- do.call(rbind, svs)
-svs <- svs[!duplicated(svs[, 1]), ]
+svs <- unique(svs)
 
 # Confirm correct dimensions
 if (length(colnames(svs)) != 4) {
     stop("SV file must have 4 columns with sample_id and MYC, BCL2, and BCL6 SV status. ")
 }
 
+# Confirm sample_id column
```

Comment:
I'm confused how this would work. If the sample ID column is named SV.SAMPLE_ID (and renamed later) wouldn't this check always fail?

Reply:
The input table has to have the column named "sample_id" for the later renaming steps to work, so all this check does is confirm that that column exists and is named correctly. 

This is all awkwardly done to avoid loading in tidyverse...